components/infra-proxy-service/migrations/pipeline/utility.go

```diff
@@ -341,3 +342,22 @@ func Unzip(ctx context.Context, mst storage.MigrationStorage, result pipeline.Re
 	}
 	return result, nil
 }
+
+func ValidateZip(ctx context.Context, st storage.Storage, mst storage.MigrationStorage, result pipeline.Result) (pipeline.Result, error) {
+	var err error
+	unzipFolder := result.Meta.UnzipFolder
+
+	_, err = os.Stat(unzipFolder + "/organizations")
```

Comment:
We should be implementing it this way:
1. Find under unzip folder where the organisations folder exist. So if it is under <unzip_folder>/backup_20220221/new_backup/organisations
2. Set the location in results.Meta.UnzipFolder. Here it should be result.Meta.UnzipFolder = <unzip_folder>/backup_20220221/new_backup
3. Check key_dump.json in the same location so we expect key_dump.json under <unzip_folder>/backup_20220221/new_backup